Dockerfile

```diff
@@ -0,0 +1,155 @@
+# --------------------------------------------------------
+# 1) Base Image
+# --------------------------------------------------------
+FROM ubuntu:22.04
+
+LABEL maintainer="Santhosh Janardhanan <santhoshj@gmail.com>" \
+    description="Dockerfile to build and run the latest OpenTTD from source."
+ARG APPVER=14.1
```

Comment:
You cannot hardcode a version and call it 'Latest'. As soon the hardcoded version won't be 'Latest' any more.